### PR TITLE
Initial project structure based on cookiecutter-data-science

### PR DESCRIPTION
**Pull request by onomatopeia**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+The MIT License (MIT)
+Copyright (c) 2020, agata
```
Comment (eMaerthin) [inline LICENSE]: `Copyright (...) agata` :D, maybe we should use a name of our group instead. I don't know it yet though. The name of the group will be communicated on Monday.
wdyt?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+clean:
+	find . -type f -name "*.py[co]" -delete
+	find . -type d -name "__pycache__" -delete
+
+## Lint using flake8
+lint:
+	flake8 src
+
+## Upload Data to S3
+sync_data_to_s3:
```
Comment (eMaerthin) [inline Makefile]: I am not sure if `aws` will be used, IMO we will not need it in the first part of the project, anyway we can left it here for now:)

File: README.md [hunk 3]
```diff
@@ -13,3 +13,52 @@
 ### Tracking the epidemics
 * [Timeline of the 2019 Wuhan coronavirus outbreak](https://en.wikipedia.org/wiki/Timeline_of_the_2019%E2%80%9320_Wuhan_coronavirus_outbreak)
 * [Tracking coronavirus: Map, data and timeline](https://bnonews.com/index.php/2020/02/the-latest-coronavirus-cases/)
+
+Project Organization
```
Comment (eMaerthin) [inline README.md]: please make header here
```suggestion
## Project Organization
```

File: LICENSE (hunk 1 above)
Comment (onomatopeia) [reply LICENSE]: changed to a temporary name that describes the group. will rename again when the official name is known

**Pull request merged**